source/includes/_endpoint-multitable.md

```diff
@@ -0,0 +1,88 @@
+## Multitables
+
+### Catalog
+
+`/datasets/{dataset_id}/multitables/`
+
+#### GET
+
+```json
+{
+    "element": "shoji:catalog",
+    "self": "/api/datasets/123/multitables/",
+    "specification": "/api/specifications/multitables/",
+    "description": "List of multitable definitions for this dataset",
+    "index": {
+        "/api/datasets/123/multitables/7ab1e/": {
+            "is_public": false,
+            "owner_id": "/api/users/b055/",
+            "name": "Basic Demographics",
+            "id": "7ab1e"
+        }
+    }
+}
+```
+
+GET on this resource returns a Shoji Catalog with the list of Multitables that the
+current user can use on this Dataset.
+
+This index contains two kinds of multitables: those that belong to the dataset, denoted by the `is_public` tuple attribute; and those that belong to the current user. Personal multitables are those created by the authenticated user, and they cannot be accessed by other users. Dataset multitables are available to all users who are authorized to view the dataset.
+
+#### POST
+
+POST a Shoji Entity to this catalog to create a new multitable definition. Entities must include a `name` and `template`; the template must contain a series of objects with keys `variable` and `query` and optionally `transform`. If omitted, `is_public` defaults to `false`. A successful POST yields a 201 response
+that will contain a Location header with the URL of the newly created multitable.
+
+All users with access to the dataset can create personal multitable definitions; however, only the current dataset editor can create public multitables (`is_public: true`) which everyone with access to the dataset can see. Attempting to create a public multitable when not the current dataset editor results in a 403 response.
+
+#### PATCH
+
+There are no elements of the catalog that can be changed via PATCH.
```

Comment:
FWIW this is not consistent with other catalogs, which at least let you edit names.